app/components/ui/Cards/CategoryCard.module.scss

```diff
@@ -0,0 +1,59 @@
+@import "../../../styles/utils/_helpers.scss";
+
+.categoryCard {
+  display: flex;
+  flex-direction: row;
+  gap: $general-spacing-s;
+  background-color: $white;
+  padding: $general-spacing-lg;
+  height: auto;
+  width: 100%;
+  border-radius: $rounded-md;
+  box-shadow: 0px 1px 3px 0px rgba(112, 79, 29, 0.1),
+    0px 1px 2px -1px rgba(112, 79, 29, 0.1);
+  position: relative;
+  cursor: pointer;
+
+  @media screen and (min-width: $break-tablet-p) {
+    flex-direction: column;
+    height: 150px;
+  }
+
+  &:hover {
+    color: $color-brand-dark;
+  }
+
+  .categoryTitle a {
+    font-size: 1.25rem;
+    font-weight: 600;
+    color: $text-primary;
+    transition: none;
+
+    @media screen and (max-width: $break-tablet-p) {
+      font-size: 1.125rem;
+    }
+
+    &:hover {
+      color: $color-brand-dark;
+      text-decoration: none;
+    }
+
+    &::after {
+      content: "";
+      position: absolute;
+      left: 0;
+      top: 0;
+      right: 0;
+      bottom: 0;
+    }
+  }
+
+  // FontAwesome Icon Styling
+  span {
```

Comment:
Can we target the icon using the .fas class instead? Just in case another span were to appear in the future.

Reply:
I added an icon class name to the span instead to follow the current css module pattern (ex. styles.fas wouldn't work)

Reply:
Can you clarify? I don't see the change in the scss file.

Reply:
Im not sure why its not reflected in the Files changes tab, but you can see the .icon class if you view the [full file](https://github.com/Exygy/askdarcel-web/blob/bac606d23f551fd14a84f58a3bbcad3f3d94429e/app/components/ui/Cards/CategoryCard.module.scss)